apps/src/code-studio/stageLockRedux.js

```diff
@@ -110,6 +119,12 @@ export default function reducer(state = initialState, action) {
 }
 
 // Action creators
+
+/**
+ * Authorizes the user to be able to see lockable stages
+ */
+export const authorizeLockable = () => ({ type: AUTHORIZE_LOCKABLE });
```

Comment:
Nit: Top-level arrow functions / function expressions.  Airbnb says [prefer function declarations over function expressions](https://github.com/airbnb/javascript#functions--declarations) (and by extension, arrow functions) unless you are inside a non-function block.